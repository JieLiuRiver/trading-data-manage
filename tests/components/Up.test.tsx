import React from 'react';
import { render } from '@testing-library/react';
import UpIcon from '@/components/SvgIcons/Up';

describe('UpIcon', () => {
  it('should render correctly with default props', () => {
    const { container } = render(<UpIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render correctly with size and color props', () => {
    const { container } = render(<UpIcon size={48} color="red" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
