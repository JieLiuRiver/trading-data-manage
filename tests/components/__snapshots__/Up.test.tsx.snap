// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpIcon should render correctly with default props 1`] = `
<svg
  class="icon"
  height="24"
  p-id="8939"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M859.062857 256l-170.788571 20.48a6.948571 6.948571 0 0 0-4.022857 11.702857l50.834285 49.005714-193.462857 193.828572-87.405714-87.405714a13.897143 13.897143 0 0 0-19.382857 0l-275.748572 275.748571a7.314286 7.314286 0 0 0 0 9.874286L197.851429 768a6.948571 6.948571 0 0 0 9.508571 0l237.348571-236.982857 87.04 87.04a14.262857 14.262857 0 0 0 19.382858 0l232.594285-232.594286 50.834286 50.834286a6.948571 6.948571 0 0 0 11.702857-4.022857l20.48-170.788572a6.582857 6.582857 0 0 0-7.68-5.485714z"
    p-id="8940"
  />
</svg>
`;

exports[`UpIcon should render correctly with size and color props 1`] = `
<svg
  class="icon"
  fill="red"
  height="48"
  p-id="8939"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="48"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M859.062857 256l-170.788571 20.48a6.948571 6.948571 0 0 0-4.022857 11.702857l50.834285 49.005714-193.462857 193.828572-87.405714-87.405714a13.897143 13.897143 0 0 0-19.382857 0l-275.748572 275.748571a7.314286 7.314286 0 0 0 0 9.874286L197.851429 768a6.948571 6.948571 0 0 0 9.508571 0l237.348571-236.982857 87.04 87.04a14.262857 14.262857 0 0 0 19.382858 0l232.594285-232.594286 50.834286 50.834286a6.948571 6.948571 0 0 0 11.702857-4.022857l20.48-170.788572a6.582857 6.582857 0 0 0-7.68-5.485714z"
    p-id="8940"
  />
</svg>
`;
